compiler/test/suites/basic_functionality.re

```diff
@@ -244,7 +244,11 @@ describe("basic functionality", ({test, testSkip}) => {
     "Failure: boo",
   );
   assertSnapshotFile("toplevel_statements", "toplevelStatements");
-  assertSnapshotFile("unsafe_wasm_globals", "unsafeWasmGlobals");
+  assertSnapshotFile(
+    ~config_fn=() => {Grain_utils.Config.no_gc := true},
```

Comment:
why do we set the config_fn for things like this instead of changing the source file to add the disableGC attribute?

Reply:
I believe disabling GC is something we want to be doing on the level of the whole program rather than on a per-file basis right @ospencer ?